modules/ROOT/partials/integrations/react-bundling.adoc

```diff
@@ -35,9 +34,8 @@ import 'tinymce/plugins/table';
 import 'tinymce/plugins/help';
 
 // Content styles, including inline UI like fake cursors
-/* eslint import/no-webpack-loader-syntax: off */
-import contentCss from '!!raw-loader!tinymce/skins/content/default/content.min.css';
-import contentUiCss from '!!raw-loader!tinymce/skins/ui/oxide/content.min.css';
+import contentCss from 'tinymce/skins/content/default/content.min.css?inline';
+import contentUiCss from 'tinymce/skins/ui/oxide/content.min.css?inline';
```

Comment:
we have `js` files now that are better for bundling?

Reply:
I've swapped it to import the skin `js` files directly